config/routes.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :clients, only: %i[index show]
-  get "/clients/eligibility", to: "clients#eligibility", constraints: { format: 'json' }
+  resources :clients, only: %i[index show] do
+    get 'eligibility/:day', to: 'clients#eligibility', constraints: { format: 'json' }
+  end
```

Comment:
How does `client_id` get into here? Wouldn't it need to be `get ':client_id/eligibility/:day'`? Or do I misunderstand how this block works?

Reply:
rails routing does that does us, when we next it under the line above.

Reply:
👍